jupyterlab_nodeeditor/backup_framework.py

```diff
@@ -0,0 +1,91 @@
+# TO-DO
```

Comment:
If these are still valid and necessary for the PR to be accepted, can you make them into todo items in the PR body?